modules/lang-painless/src/main/java/org/elasticsearch/painless/lookup/PainlessLookup.java

```diff
@@ -163,18 +163,6 @@ public PainlessMethod lookupPainlessMethod(Class<?> targetClass, boolean isStati
         return lookupPainlessObject(targetClass, objectLookup);
     }
 
-    public List<PainlessMethod> lookupPainlessSubClassesMethod(String targetCanonicalClassName, String methodName, int methodArity) {
```

Comment:
Not strictly necessary here but these appear unused so I dropped them here as they're not really worth a separate PR I figured?

Reply:
@jdconrad Do you know if this method was refactored away and forgotten, or if it is a bug that it is unused?

Reply:
These have never been used. There were plans to use these when refactoring portions of the AST, but they never came to fruition. I'm good with removing these now.